scripts/nayduck.py

```diff
@@ -67,13 +70,28 @@ def _parse_args():
     return args
 
 
-def get_curent_sha():
-    return subprocess.check_output(['git', 'rev-parse', 'HEAD'], text=True)
+def get_sha(branch: str):
+    try:
+        sha = subprocess.check_output(['git', 'rev-parse', branch], text=True)
+
+    except subprocess.CalledProcessError as e:
+        remote_branch = 'remotes/origin/' + branch
+        print(
+            f"Couldn't find a local branch \'{branch}\'. Trying remote: {remote_branch}"
+        )
+        sha = subprocess.check_output(['git', 'rev-parse', remote_branch],
+                                      text=True)
```

Comment:
nit: you can add a comma at the end of the list and after text for a bit nicer formatting (in my opinion ;) )